### PR TITLE
content updates for issue 108

### PR DESCRIPTION
**Pull request by weiwang-gsa**

### CODE REVIEW

File: training-front-end/src/content/training_travel/lesson03.mdx [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
-title: "Lesson 3: Obtaining a Travel Account"
+title: "Lesson 3: Obtaining a Travel card/account"
```
Comment (mark-meyer) [inline training-front-end/src/content/training_travel/lesson03.mdx:2]: I think Andrew _just_ changed this so the capitalization would be consistent with Lesson 5. Should now be `Card/Account`.

**Review by mark-meyer — COMMENTED** [review]
This looks good to me. There is just one small change Andrew made after you submitted it.

File: training-front-end/src/content/training_travel/lesson03.mdx (hunk 1 above)
Comment (weiwang-gsa) [reply training-front-end/src/content/training_travel/lesson03.mdx:2]: Thank you so much, will make the change and will double check the whole document to make sure there is no other new changes

**Review by mark-meyer — APPROVED** [review]
Looks good to me!

**Pull request merged**